### PR TITLE
os: Adding booting-on-packet

### PR DESCRIPTION
**Pull request by stresler**

### CODE REVIEW

File: os/booting-on-packet.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  units:
+    - name: etcd2.service
+      command: start
+    - name: fleet.service
+      command: start
+```
+
+## IP Addresses
+
+The `$private_ipv4`, `$public_ipv4`, and `private_ipv6` variables are fully supported in cloud-config on Packet. Packet is fully IPv6 compliant and we encourage you to utilize IPv6 for public connectivity with your running containers. Make sure to read up on [IPv6 and Docker](https://docs.docker.com/articles/networking/#ipv6) if you choose to take advantage of this functionality.
```
Comment (robszumski) [inline os/booting-on-packet.md]: I believe the `$private_ipv6` variable is only supported in the DigitalOcean OEM.

Comment (robszumski) [inline os/booting-on-packet.md]: I believe the `$private_ipv6` variable is only supported in the DigitalOcean OEM.

Comment (stresler) [inline os/booting-on-packet.md]: typo, good catch.

File: os/booting-on-packet.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  units:
+    - name: etcd2.service
+      command: start
+    - name: fleet.service
+      command: start
+```
+
+## IP Addresses
+
+The `$private_ipv4`, `$public_ipv4`, and `$public_ipv6` variables are fully supported in cloud-config on Packet. Packet is fully IPv6 compliant and we encourage you to utilize IPv6 for public connectivity with your running containers. Make sure to read up on [IPv6 and Docker](https://docs.docker.com/articles/networking/#ipv6) if you choose to take advantage of this functionality.
```
Comment (robszumski) [inline os/booting-on-packet.md]: Sorry, I was a little unclear, in addition to the typo, I don't think that the variable exists at all according to our OS team. Have you found that this works in your testing?

Comment (robszumski) [inline os/booting-on-packet.md]: (exists for your OEM)

Comment (stresler) [inline os/booting-on-packet.md]: Let me test to make sure, it is created as an environment variable, but I don't know that I've tested in a cloud config to be honest.  It should be working, but let me confirm.

Comment (stresler) [inline os/booting-on-packet.md]: This is where it should be subbed https://github.com/coreos/coreos-cloudinit/blob/master/initialize/env.go#L50 and here is where I create it https://github.com/coreos/coreos-cloudinit/blob/master/datasource/metadata/packet/metadata.go#L89 

So, it should work, but let me run a test provision to be certain.

Comment (robszumski) [inline os/booting-on-packet.md]: Ah ok, looks good. Let me know how the test goes.

**Comment by robszumski** [comment]
LGTM other than testing that `$public_ipv6` works.

File: os/booting-on-packet.md (hunk 2 above)
Comment (stresler) [inline os/booting-on-packet.md]: Yup, confirmed working.

**Comment by stresler** [comment]
Curious - how often do the docs release?

**Comment by robszumski** [comment]
They'll get sucked in with the next alpha release (~weekly)

**Comment by robszumski** [comment]
Live! https://coreos.com/os/docs/latest/booting-on-packet.html

**Pull request merged**